docs/table_persistence.md

```diff
@@ -0,0 +1,123 @@
+                                                # UCX Persistence
+
+Enumeration of all UCX persistence elements
+
+## Overview
+Table Utilization:
+
+| Table              | Generate Assessment | Migrate Local Groups | Migrate External Tables | Migrate SQL Warehouses | Upgrade Jobs | Migrate managed tables |
+|--------------------|---------------------|----------------------|-------------------------|------------------------|--------------|------------------------|
+| tables             | RW                  |                      | RO                      |                        |              | RO                     |
+| grants             |                     |                      | RW                      |                        |              | RW                     |
+| mounts             | RW                  |                      | RO                      |                        | RO           | RO                     |
+| permissions        |                     | RW                   |                         |                        |              |                        |
+| jobs               | RW                  |                      |                         |                        | RO           |                        |
+| clusters           | RW                  |
+| external_locations | RW                  |                      | RO                      |
+| workspace          | RW                  | RO                   |                         |                        | RO           |
+
+**RW** - Read/Write the job that generates the table
+**RO** - Read Only
+
+### Inventory Database
+#### _$inventory_.tables
+
+Holds Inventory of all tables in all databases and their relevant metadata.
+
+| Column    | Datatype | Description | Comments |
+|-----------|----------|-------------|----------|
+| catalog   | string   | Original catalog of the table. _hive_metastore_ by default |
+| database  | string   | Original schema of the table |
+| name |string|Name of the table|
+|object_type|string|MANAGED, EXTERNAL, or VIEW|
+|table_format|string|Table provider. Like delta or json or parquet.|
+|location|string|Location of the data for table|
+|view_text|nullable string|If the table is the view, then this column holds the definition of the view|
+|workspace_id|string|A workspace ID of the HMS (Optional)|
```

Comment:
```suggestion
```
remove this :) it's going to be added on the account level - see https://github.com/databrickslabs/ucx/pull/299/files